cmd/get/clusters/printer.go

```diff
@@ -29,6 +29,8 @@ func (r *runner) printOutput(clusterResource cluster.Resource) error {
 			resource = provider.GetAzureTable(clusterResource)
 		case key.ProviderOpenStack:
 			resource = provider.GetOpenStackTable(clusterResource)
+		default:
+			resource = provider.GetCommonClusterTable(clusterResource)
```

Comment:
The `get clusters` command was failing for unknown providers. Added a common cluster table rendering and applied it to all existing providers, which do not require specific handling, as well as unknown providers